pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -22,10 +22,10 @@ stdenv.mkDerivation rec {
   # When updating, use https://github.com/systemd/systemd-stable tree, not the development one!
   # Also fresh patches should be cherry-picked from that tree to our current one.
   src = fetchFromGitHub {
-    owner = "NixOS";
+    owner = "abbradar";
```

Comment:
Can you make a PR to the NixOS/systemd repo instead of pointing to abbradar/systemd?

Reply:
I'm still testing, the first iteration of the patch didn't work. I'll do that right as my tests pass!